digits/model/tasks/caffe_train.py

```diff
@@ -39,6 +39,50 @@
 CAFFE_DEPLOY_FILE = 'deploy.prototxt'
 
 @subclass
+class DigitsTransformer(caffe.io.Transformer):
```

Comment:
Ok yeah, I guess this is fine. The only problem would be if `preprocess()` had a `resize` argument. But if the class has a `resize` argument then we're ok. Nevermind!